demo/src/sandboxes/animating-auto/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "noImplicitThis": true,
     "noImplicitAny": true,
     "strictNullChecks": true,
-    "suppressImplicitAnyIndexErrors": true,
```

Comment:
NEED REVIEWER HERE

- [ ] removed suppressImplicitAnyIndexErrors from ts to get rid of this error

<img width="745" alt="image" src="https://github.com/user-attachments/assets/edd1d35f-1872-4651-867b-97378c8f1e88" />

Reply:
cool 👍🏼

Reply:
Assuming it is resolved. Don't hesitate to reopen if it is not.